lib/std/start.zig

```diff
@@ -116,6 +116,21 @@ fn _start() callconv(.Naked) noreturn {
                 : [argc] "=r" (-> [*]usize)
             );
         },
+        .powerpc64le => {
+            // Before returning the stack pointer, we have to set up a backchain
+            // and a few other registers required by the ELFv2 ABI.
+            // TODO: Support powerpc64 (big endian) on ELFv2.
```

Comment:
From my reading of the ABI spec, it seems that there's no difference in stack frame setup between big and little endian modes.
Or am I missing something here?

Reply:
The distinction comes from the fact that there are two separate ABIs (ELFv1 and ELFv2) in use. Little endian distros exclusively use ELFv2, but big endian distros can use either. This patchset only supports ELFv2, so in order to support big endian here, an additional check ensuring that the ELFv2 ABI is in use would be required.